### PR TITLE
Docker integration

### PR DESCRIPTION
**Pull request by Soullivaneuh**
To provide a "ready to use" dev env.

### CODE REVIEW

File: docker/conf/php.ini [hunk 1]
```diff
@@ -0,0 +1,8 @@
+; PHP.ini required and recommended options.
+; Used by GitLab-CI and docker-compose
+
+[PHP]
+memory_limit = -1
+
+[Date]
+date.timezone = Europe/Paris
```
Comment (greg0ire) [inline docker/conf/php.ini]: Not sure about this :P

Comment (Soullivaneuh) [inline docker/conf/php.ini]: Well, I set this to not get PHP yelling.

But this can indeed be great if it can be configurable.

Any suggestion?

**Comment by greg0ire** [comment]
Utc?

**Comment by Soullivaneuh** [comment]
> Utc?

Well, depends of where you are, right? :stuck_out_tongue:

**Comment by greg0ire** [comment]
Utc means Coordinated **Universal** Time, so no, it does not depend, it applies to the whole Earth (and even the universe ?).

**Pull request merged**